README.md

```diff
@@ -175,10 +175,10 @@ For Maven builds, the version of the Gradle Enterprise Maven extension automatic
 
 | TC Build Scan plugin version | Injected GE Maven extension version | Injected Common CCUD Maven extension version |
 |------------------------------|-------------------------------------|----------------------------------------------|
-| Next                         | 1.17.2                              | 1.11.1                                       |
-| 0.33                         | 1.16.1                              | 1.11.1                                       |
-| 0.32                         | 1.15.4                              | 1.11.1                                       |
-| 0.31                         | 1.15.3                              | 1.11.1                                       |
+| Next                         | 1.17.2                              | 1.12                                         |
+| 0.33                         | 1.16.1                              | 1.12                                         |
+| 0.32                         | 1.15.4                              | 1.12                                         |
+| 0.31                         | 1.15.3                              | 1.12                                         |
```

Comment:
Only next should change to 1.12.

Reply:
I'll fix it.

Reply:
Fixed.

Reply:
Oops. Good catch. This makes total sense why, but it did not occur to me at the time.

I'll add a note to the version upgrade process doc to watch out for this as right now it simply states "Do a global find and replace of the current version with the new version".